java/fury-core/src/main/resources/fury/blacklist.txt

```diff
@@ -1,151 +1,246 @@
-clojure.core$constantly
-clojure.main$eval_opt
-org.codehaus.groovy.runtime.MethodClosure
+bsh.Interpreter
+bsh.XThis
+ch.qos.logback.core.db.DriverManagerConnectionSource
+ch.qos.logback.core.db.JNDIConnectionSource
+clojure.core
+clojure.main
+com.caucho.config.types.ResourceRef
 com.caucho.hessian.test.TestCons
+com.caucho.naming.QName
+com.ibm.jtc.jax.xml.bind.v2.runtime.unmarshaller.Base64Data
+com.ibm.xltxe.rnm1.xtq.bcel.util.ClassLoader
+com.mchange.v2.c3p0.impl.PoolBackedDataSourceBase
 com.mchange.v2.c3p0.JndiRefForwardingDataSource
 com.mchange.v2.c3p0.WrapperConnectionPoolDataSource
+com.mysql.cj.jdbc.MysqlConnectionPoolDataSource
+com.mysql.cj.jdbc.MysqlDataSource
+com.mysql.cj.jdbc.MysqlXADataSource
+com.mysql.jdbc.jdbc2.optional.MysqlDataSource
+com.mysql.jdbc.util.ServerController
 com.rometools.rome.feed.impl.EqualsBean
 com.rometools.rome.feed.impl.ToStringBean
+com.sun.corba.se.impl.activation.ServerManagerImpl
+com.sun.corba.se.impl.activation.ServerTableEntry
+com.sun.corba.se.impl.presentation.rmi.InvocationHandlerFactoryImpl.CustomCompositeInvocationHandlerImpl
+com.sun.corba.se.spi.orbutil.proxy.CompositeInvocationHandlerImpl
+com.sun.corba.se.spi.orbutil.proxy.LinkedInvocationHandler
+com.sun.jndi.ldap.LdapAttribute
 com.sun.jndi.rmi.registry.BindingEnumeration
 com.sun.jndi.toolkit.dir.LazySearchEnumerationImpl
+com.sun.org.apache.bcel.internal.util.ClassLoader
 com.sun.org.apache.xalan.internal.xsltc.trax.TemplatesImpl
+com.sun.org.apache.xpath.internal.objects.XString
+com.sun.org.apache.xpath.internal.XPathContext
 com.sun.rowset.JdbcRowSetImpl
+com.sun.syndication.feed.impl.EqualsBean
+com.sun.syndication.feed.impl.ObjectBean
+com.sun.syndication.feed.impl.ToStringBean
 com.sun.xml.internal.bind.v2.runtime.unmarshaller.Base64Data
+com.zaxxer.hikari.HikariConfig
+com.zaxxer.hikari.HikariDataSource
+groovy.lang.GString
+groovy.lang.PropertyValue
+groovy.util.MapEntry
+java.beans.EventHandler
+java.beans.Expression
+java.lang.ProcessBuilder
+java.lang.reflect.Proxy
+java.lang.Runtime
+java.lang.UNIXProcess
+java.net.InetAddress
+java.net.Socket
+java.net.URL
```

Comment:
I'm not sure whether we should add URL to blacklist, since the serialization is common in java